xml/System.Windows.Xps/XpsDocumentWriter.xml

```diff
@@ -265,9 +265,9 @@
         <Parameter Name="e" Type="System.Windows.Documents.Serialization.WritingPrintTicketRequiredEventArgs" />
       </Parameters>
       <Docs>
-        <param name="sender">To be added.</param>
-        <param name="e">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="sender">The source of the event.</param>
+        <param name="args">The event data.</param>
```

Comment:
```suggestion
        <param name="e">The event data.</param>
```
Because of the disparity in parameter names, "The event data" does not appear in the build.